packages/styles/accounts.import.styl

```diff
@@ -0,0 +1,55 @@
+.modal-dialog
+  +above(1)
+    width 480px
+    margin 100px auto
+
+.modal-body
+  padding 0
+
+.modal-content
+  border-radius 5px
+  box-shadow 0 5px 15px rgba(0,0,0,.3)
+
+.at-form-wrap
+  padding 25px
+
+.at-form
+  margin 0
+
+.at-title
+  display none
+  h3
+    padding 10px
+    margin 0
+    color white
+
+.at-pwd-form
+  fieldset
+    border 0
+    padding 0 20px
+
+.at-form
+  padding 0
+  border-radius 5px
+
+.at-pwd-form
+  > form
+    label
+      font-weight bold
+
+.at-signup-link
+  padding 0 20px
+  text-align center
+
+.at-btn
+  makeButton($confirm)
```

Comment:
I guess was thinking about it outside of the bounds of bootstrap—green is more appropriate for a call to action but it escapes the pattern bootstrap establishes. I'll use the primary color instead.